File: 2012/submissions/Ridhwaan Shakeel/1/Ruby (can open in terminal)/wordhunt.rb
#!/usr/bin/ruby

print "Enter user input text: "
$stdout.flush
text = gets.chomp
print "Enter word to search: "
$stdout.flush
word = gets.chomp

#puts "'" + text + "'"
#puts "'" + word + "'"

if ( text =~ /\b#{word}\b/)
  puts  "Word '" + word + "' found at position " + text.index(word).to_s
else
  puts "Word '" + word + "' not found"
end

File: 2012/submissions/Ridhwaan Shakeel/1/Perl (eclipse with perl sdk plugin)/wordhunt.pl
#!/usr/bin/perl
#use strict;
#use warnings;

print "Enter text input: ";
my $text = <>;
print "Enter a word to search: ";
my $word = <>;

chomp($word);

if ($text =~ ".*\\b$word\\b.*")
{
	my $result = index $text, $word;
	print "Word '$word' found at position $result\n";	
}
else
{
	print "Word '$word' not found";
}
